tests/test_to_numpy.py

```diff
@@ -58,6 +58,13 @@ def test_list_tuple(self):
         result = ToNumpy()(test_data)
         np.testing.assert_allclose(result, np.asarray(test_data))
 
+    def test_single_value(self):
+        for test_data in [5, np.array(5)]:
```

Comment:
suggest to add `torch.tensor(5)`

Reply:
Sure, updated in the latest commit.

Thanks.